README.md

```diff
@@ -77,33 +67,33 @@ handling logic.
 	> Note that you can use any file name or package name (convention is to make
 	package name same as directory name).
 
-1. Now go to the `cmd` subdirectory.
-	```sh
-	cd cmd
-	```
+1. To run locally, you'll need to create a main package to start your server
+   (see instructions below for container builds to skip this step and match your
+   local development environment to production):
```

Comment:
We'll probably want to split the quickstart sections into two
- Developing with your own `main.go` locally
- Developing with a Docker and buildpacks

That change can be a fast follow-up.